magento-devdocs/magento-openapi-transformer
# Refactor script for cli environments

Refactors the project to make it more friendly for CLI environments such as build pipelines.

## Testing this PR

1. Check out this branch
1. Install new dependencies: `yarn install`
1. Run script: `yarn start -i /path/to/schema/file -o /path/to/write/output`
1. Verify script runs successfully
1. Verify output file has been written
